actors/crypto/randomness.go

```diff
@@ -15,42 +16,55 @@ import (
 type DomainSeparationTag int
 
 const (
-	DomainSeparationTag_TicketDrawing DomainSeparationTag = 1 + iota
-	DomainSeparationTag_TicketProduction
+	DomainSeparationTag_TicketProduction DomainSeparationTag = 1 + iota
 	DomainSeparationTag_ElectionPoStChallengeSeed
 	DomainSeparationTag_SurprisePoStChallengeSeed
 	DomainSeparationTag_SurprisePoStSelectMiners
 	DomainSeparationTag_SurprisePoStSampleSectors
 )
 
 // Derive a random byte string from a domain separation tag and the appropriate values
-func DeriveRandWithMinerAddr(tag DomainSeparationTag, tix abi.RandomnessSeed, minerAddr addr.Address) abi.Randomness {
+func DeriveRandWithMinerAddr(tag DomainSeparationTag, randSeed []byte, minerAddr addr.Address) []byte {
```

Comment:
`abi.Randomness` still exists, why doesn't this return one? I think there is some legit type safety on offer here with those types.